test/extensions/filters/http/ratelimit/config_test.cc

```diff
@@ -81,11 +81,12 @@ TEST(RateLimitFilterConfigTest, BadRateLimitFilterConfig) {
                           "route_key: Cannot find field");
 }
 
-// Test that the deprecated extension name still functions.
+// Test that the deprecated extension name is disabled by default.
```

Comment:
So I'm fine with all of these testing failure by default but if we hit coverage issue we may want to TestScopedRuntime these back to the prior behavior

Reply:
I think the coverage is fine because there's still a test that exercises the logging path (e.g. when the runtime flag is enabled).